code/app/src/main/java/ca/ualberta/ishelf/BorrowFragment.java

```diff
@@ -122,6 +164,169 @@ public boolean onQueryTextChange(String text) {
         return view;
     }
 
+
+
+    public void Filter(String filter, String object){
+        if(object.equals("Book")){
+            if(filter.equals("1")) {
```

Comment:
if (book.getAveRating()>=filter.toInt(){
    ratedBooks.add(book);
}

Reply:
can do that while optimizing the code later, for now lets just keep this